### DISCUSSION TITLE
How to update pnpm?

### DISCUSSION THREAD

**Discussion by MartijnHarmenzon** [pnpm/pnpm#5310] [Q&A]
![image](https://user-images.githubusercontent.com/23123158/188594345-300a715b-ffc4-43c9-8e55-cf65643fa76e.png)

Executing `pnpm add -g pnpm` does not work. It stays in version 7.1.6. How to resolve?

**Comment by ntnyq** [comment]
`pnpm add -g @pnpm/exe`

**Reply by MartijnHarmenzon** [reply]
Thanks @ntnyq . Unfortunately, it didn't work for me.

![image](https://user-images.githubusercontent.com/23123158/189064904-963be8d4-015d-4675-8ec9-d13c487e9cad.png)
